opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -851,6 +863,18 @@ def save_as_onnx(
 
         # save tokenizer.json in output_path
         model.save(save_json_folder_path)
+        tokenizer_file_path = os.path.join(save_json_folder_path, "tokenizer.json")
```

Comment:
Dear @dhrubo-os , I think no need to change the unit test in order to avoid duplicating. 
My recently implemented reusable ```fix_truncation``` function is used in ```save_as_pt``` and ```save_as_onnx``` in the same way. So, ```test_truncation_parameter``` unit test checks the work of ```fix_truncation``` in ```save_as_pt``` function.
I think we should leave it as it's already implemented. But I will be happy to see your suggestions;)